pkg/commands/git_commands/branch_test.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/go-errors/errors"
 	"github.com/jesseduffield/lazygit/pkg/commands/oscommands"
+	"github.com/jesseduffield/lazygit/pkg/config"
```

Comment:
tests aren't directly related to other changes in this PR but worth including

Reply:
In that case my preference would be to have them in a separate "increase test coverage" commit (before the other one).